docs/modules/usage/prompting/customization.md

```diff
@@ -0,0 +1,67 @@
+# Customizing Agent Behavior
+
+OpenHands can be customized to work more effectively with specific repositories by providing repository-specific context and guidelines. This section explains how to optimize OpenHands for your project.
+
+## Repository Configuration
+
+You can customize OpenHands' behavior for your repository by creating a `.openhands` directory in your repository's root. At minimum, it should contain the file
```

Comment:
```suggestion
You can customize OpenHands' behavior for your repository by creating a `.openhands` directory in your home directory. At minimum, it should contain the file
```

Reply:
This is a different dir 🙃 I moved the homedir one to `.openhands-state` for clarity